Added json rpc spec page (#1244)
iotaledger/iota-wiki: docs/build/wasp-evm/0.7/docs/reference/json-rpc-spec.md

# JSON-RPC API

> [JSON-RPC](https://www.jsonrpc.org/specification) is a stateless, light-weight remote procedure call (RPC) protocol. It defines several data structures and the rules around their processing. It is transport agnostic in that the concepts can be used within the same process, over sockets, over HTTP, or in many various message passing environments. It uses JSON (RFC 4627) as data format.

Reviewer: ```suggestion
The [JSON-RPC](https://www.jsonrpc.org/specification) is a stateless, lightweight remote procedure call (RPC) protocol. It defines several data structures and the rules around their processing. It is transport agnostic because the concepts can be used within the same process, over sockets, HTTP, or in various message-passing environments. It uses JSON (RFC 4627) as data format.
```

> This page deals mainly with the JSON-RPC API used by EVM execution clients.

Reviewer: ```suggestion
This page deals with the JSON-RPC API used by EVM execution clients.
```

| 2   | [web3_sha](https://openethereum.github.io/JSONRPC-web3-module#web3_sha3)                     | _Returns Keccak-256 (not the standardized SHA3-256) of the given data_ | ✅ |

## JSON-RPC methods according to the [Net Module API](https://openethereum.github.io/JSONRPC-net-module)

| Num | Method         | Description                                                            | Status                         |
|-----|----------------------------------------------------------------------------------|------------------------------------------------------------------------|--------------------------------|
| 1   | [net_listening](https://openethereum.github.io/JSONRPC-net-module#net_listening) | _Returns true if client is actively listening for network connections_ | ✅ |
| 2   | [net_peerCount](https://openethereum.github.io/JSONRPC-net-module#net_peercount) | _Returns number of peers currently connected to the client_             | ✅ |
| 3   | [net_version](https://openethereum.github.io/JSONRPC-net-module#net_version)     | _Returns the current network protocol version_                        | ✅ |

> You can read on complete set of available Spec [here](https://ethereum.github.io/execution-apis/api-documentation/).

Reviewer: ```suggestion
You can find the complete set of available specs in the [Ethereum API Documentation](https://ethereum.github.io/execution-apis/api-documentation/).
```
